src/parsing/css-translation-parsing.js

```diff
@@ -19,7 +19,9 @@
       if (coords.length != 3) {
         return null;
       }
-      return [new CSSTranslation(coords[0], coords[1], coords[2]), remaining];
+      var result = [new CSSTranslation(coords[0], coords[1], coords[2]), remaining];
+      result[0]._inputType = '3';
```

Comment:
> I think that might be more confusing, and add a lot of code. What about if we just set _cssText directly from the parser? That gets rid of the magic values.

Actually I can't completely get rid of the magic flags, but I won't have to store them. And I can put them in an enum.